base/uk.ac.stfc.isis.ibex.nicos/src/uk/ac/stfc/isis/ibex/nicos/messages/QueueScript.java

```diff
@@ -39,8 +40,14 @@
      *            code in script
      */
     public QueueScript(String name, String code) {
-        this.name = name;
-        this.code = code;
+        this.command = "queue";
+        this.parameters = Arrays.asList(name, code);
+    }
+
+    @Override
+    public ReceiveMessage parseResponse(String response) throws ConversionException {
+        JsonDeserialisingConverter<String> deserial = new JsonDeserialisingConverter<>(String.class);
+        return new ReceiveStringMessage(deserial.convert(response));
```

Comment:
Sort of, the response that comes from the server will be a JSON string and so will have quotation marks in and potentially some whitespace, this will strip those.